package.json

```diff
@@ -74,6 +74,7 @@
     "react-redux": "^4.4.5",
     "react-router": "^2.5.2",
     "react-router-redux": "^4.0.5",
+    "reactstrap": "^5.0.0-beta",
```

Comment:
Why did you choose a beta over a stable version?

Reply:
This matches what we use in silverstripe/admin =)